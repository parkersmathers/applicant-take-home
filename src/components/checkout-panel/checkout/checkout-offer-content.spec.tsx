import { render } from '../../../testing/test-utils';
import React from 'react';
import '@testing-library/jest-dom';

import CheckoutOfferContent from './checkout-offer-content';

describe('Test Checkout Offer Content Component', () => {
    // Matches snapshot
    test('Component matches snapshot', () => {
        const { asFragment } = render(<CheckoutOfferContent cost={4900} value={5000} bonus={2.04} />);

        expect(asFragment()).toMatchSnapshot();
    });
});
