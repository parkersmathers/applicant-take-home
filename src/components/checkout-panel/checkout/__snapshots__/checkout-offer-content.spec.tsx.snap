// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Checkout Offer Content Component Component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="offer-content"
  >
    <div
      class="offer-content__row"
    >
      <span>
        Redemption Amount:
      </span>
       
      <span>
        $49.00
      </span>
    </div>
    <div
      class="offer-content__row offer-content--primary"
    >
      <span>
        Prizeout Bonus (+2.04%):
      </span>
       
      <span>
        $1.00
      </span>
    </div>
    <div
      class="offer-content__row"
    >
      <span>
        You Get:
      </span>
       
      <span>
        $50.00
      </span>
    </div>
  </div>
</DocumentFragment>
`;
